specification/videoanalyzer/resource-manager/Microsoft.Media/preview/2021-05-01-preview/Videos.json

```diff
@@ -0,0 +1,1033 @@
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "NoContent"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/AzureVideoAnalyzerAccountName"
+          },
+          {
+            "name": "videoName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Video name."
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Deletes a video entity.": {
+            "$ref": "examples/video-delete.json"
+          }
+        }
+      },
+      "patch": {
+        "summary": "Updates individual properties of an existing video resource.",
+        "description": "Updates individual properties of an existing video resource.",
+        "operationId": "Videos_Update",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/VideoEntity"
+            }
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/AzureVideoAnalyzerAccountName"
+          },
+          {
+            "name": "videoName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Video name."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/VideoEntity"
+            },
+            "description": "The request parameters"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          }
+        ],
+        "x-ms-examples": {
+          "Update video entity.": {
+            "$ref": "examples/video-patch.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/videoAnalyzers/{accountName}/videos/{videoName}/streamingAuthorization": {
```

Comment:
>streamingAuthorization [](start = 146, length = 22)

This doesn't follow the ARM segment naming pattern. It should either be:
`POST /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/videoAnalyzers/{accountName}/videos/{videoName}/getStreamingAuthorization`
OR
`GET /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/videoAnalyzers/{accountName}/videos/{videoName}/streamingAuthorizations/default` (instread of `default` you can also use `latest`, `current`, `main`)

Reply:
@majastrz  we have changed these to post calls with List word in the name